WordPress/src/main/res/layout/jetpack_badge_footer.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/jetpack_badge_footer"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingBottom="@dimen/jetpack_badge_footer_bottom_spacing">
+
+    <include
+        android:id="@+id/jetpack_badge"
+        layout="@layout/jetpack_badge"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+</FrameLayout>
```

Comment:
The main reason for creating a separate layout file is because we're adding this component programmatically as footer view in the preferences list views of App Settings ([source](https://github.com/wordpress-mobile/WordPress-Android/blob/e65458e5810f57350c58d99f8b038b0985377c48/WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java#L249)) and Notification Settings([source](https://github.com/wordpress-mobile/WordPress-Android/blob/68c9c0b1478972dc6536143dc83ec00a28d1faf2/WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java#L230)).

In both cases the padding is the same, hence my reasoning that it's cleaner with another XML than by programatically adding the margin 2 times.

If it would have been only to add padding and we could have used an `<include>` I agree that would be cleaner 👍 .

Reply:
> we're adding this component programmatically as footer view in the preferences list views of App Settings ([source](https://github.com/wordpress-mobile/WordPress-Android/blob/e65458e5810f57350c58d99f8b038b0985377c48/WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java#L249)) and Notification Settings([source](https://github.com/wordpress-mobile/WordPress-Android/blob/68c9c0b1478972dc6536143dc83ec00a28d1faf2/WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java#L230)).

I searched `jetpack_badge_footer` and saw it's being used only in `me_fragment`. I didn't notice they're also being used in App Settings and Notification Settings. Now that makes sense. 😄
Good work! 👍🏻